### ISSUE TITLE
Build UI does not recognize the actual app being built

### ISSUE THREAD

**Issue by r0ckarong** [reisxd/revanced-builder#186]
When building WarnWetter, at the end of the process the header of the last page says "ReVanced has been built.". This is kinda confusing because I associate ReVanced with YouTube. Maybe add some info like "ReVanced {APP} has been built" and replacing {APP} with whatever thing you actually patched.

Also, the Download button and text in the "console" points at `revanced/revanced.apk` but this isn't correct for WetterWarn.
It should be `revanced/warnapp.apk`.

Please adjust the checks of "what" is being built and fix the prompts and locations accordingly.

**Comment by r0ckarong** [comment]
I just tested this some more. The entire steps after the "Patching" screen appears should be reworked.

The console log should only show stuff relevant to the selected app.
It will always show messages related to revanced.apk and microg.apk even if you build a different app that should result in neither one of these packages. It correctly stores the built APKs that I have tested (frontpage,warnapp,revanced) but the messages in the log console are always about `revanced`.
